src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.left = '50%';
+spider.style.top = '50%';
+spider.style.transform = 'translate(-50%, -50%)';
```

Comment:
The 'transform' property should be used with the 'translate' function to adjust the position of the spider, but first calculate the correct position.